ZukoAzula2016/.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
+encoding//src/org/team1540/zukoazula/RobotTemplate.java=UTF-8
```

Comment:
@col6y I'm 90% sure that is the setting that prevents eclipse from destroying the emoji in code. Setting the editor encoding is not a purely aesthetic change...